README.md

```diff
@@ -61,6 +61,65 @@ juju status --relations
 juju config ranger-k8s --file=user-group-configuration.yaml
 ```
 
+### Charmed OpenSearch relation
+[Charmed OpenSearch](https://charmhub.io/opensearch) should be integrated with the Ranger admin charm to enable auditing functionality for data access.
+
+Charmed OpenSearch is a machine charm, unlike Charmed Ranger which is a K8s charm. As such we will need to bootstrap a LXD controller and implement a cross-controller relation. This can be achieved by:
```

Comment:
I will make an unfounded assumption that anything that works cross-controller will work cross-model.

Also, what you suggested used to work before but I could not replicate it over the past few weeks. I think the newer versions of Juju/lxd/microk8s does not play nice with each other anymore. This can change any moment though.

Reply:
I will try cross-model when I come back to integration testing this charm. It appears they are also working on a K8s charm soon so this may be possible also to do on the same model. But to prevent blocking the relation on this test I will follow up with a separate PR for the integration test. I did eventually get cross-controller working, it was just the teardown at the end of testing which was causing the failure.